neo/Settings.cs

```diff
@@ -15,6 +15,7 @@ public class ProtocolSettings
         public IReadOnlyDictionary<TransactionType, Fixed8> SystemFee { get; }
         public Fixed8 LowPriorityThreshold { get; }
         public uint SecondsPerBlock { get; }
+        public int HashCacheSize { get; }
```

Comment:
We don't need Settings. Let's find a best value and make it fixed.

Reply:
In that case let’s try using 2x the size of the memory pool and use that.